packages/raystack/styles/reset.css

```
@@ -1,11 +1,21 @@
 /* Resets for button */
 
 button,
-[type='button'],
-[type='reset'],
-[type='submit'] {
+[type="button"],
+[type="reset"],
+[type="submit"] {
   -webkit-appearance: button;
-	appearance: button;
+  appearance: button;
```

Stylelint (17.12.0)

[error] Expected "button" to be "auto" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)